docs/elasticsearch.md

```diff
@@ -14,195 +14,489 @@ sidebar_label: Elasticsearch
 <br>
 
+## Configuration workflow
 
+1. [Install OpenEBS](/docs/next/elasticsearch.html#install-openebs)
+2. [Select OpenEBS storage engine](/docs/next/elasticsearch.html#select-openebs-storage-engine)
+3. [Configure OpenEBS Local PV StorageClass](/docs/next/elasticsearch.html#configure-openebs-local-pv-storageclass)
+4. [Installing KUDO Operator](/docs/next/elasticsearch.html#installing-kudo-operator)
+5. [Installing and Accessing Elasticsearch](/docs/next/elasticsearch.html#installing-and-accessing-elasticsearch)
+6. [Installing Kibana](/docs/next/elasticsearch.html#installing-kibana)
+7. [Installing Fluentd-ES](/docs/next/elasticsearch.html#installing-fluentd-es)
 
-## Deployment model
 
-<br>
 
+### Install OpenEBS
 
+If OpenEBS is not installed in your K8s cluster, this can be done from [here](/docs/next/installation.html). If OpenEBS is already installed, go to the next step. 
 
-<img src="/docs/assets/svg/elasticsearch-deployment.svg" alt="OpenEBS and Elasticsearch" style="width:100%;">
 
-<br>
+### Select OpenEBS storage engine
 
-<hr>
+A storage engine is the data plane component of the IO path of a Persistent Volume. In CAS architecture, users can choose different data planes for different application workloads based on a configuration policy. OpenEBS provides different types of storage engines and you should choose the right engine that suits your type of application requirements and storage available on your Kubernetes nodes. More information can be read from [here](/docs/next/overview.html#types-of-openebs-storage-engines).
 
-<br>
+After OpenEBS installation, choose the OpenEBS storage engine as per your requirement. 
 
-## Configuration workflow
+- Choose **cStor**, If you are looking for replicated storage feature and other enterprise graded features such as volume expansion, backup and restore, etc. The steps for Elasticsearch installation using OpenEBS cStor storage engine can be obtained from [here](https://github.com/openebs/cstor-operators/blob/master/docs/workload/elasticsearch/elasticsearch.md).
 
-<br>
+- Choose **OpenEBS Local PV**, If you are looking for direct attached storage or low latency data write or if the application manages data replication.
 
-1. **Install OpenEBS**
+In this document, we are deploying Elasticsearch using OpenEBS Local PV. 
 
-   If OpenEBS is not installed in your K8s cluster, this can done from [here](/docs/next/installation.html). If OpenEBS is already installed, go to the next step. 
+### Configure OpenEBS Local PV StorageClass
 
-2. **Configure cStor Pool**
+Depending on the type of storage attached to your Kubernetes worker nodes, you can select from different flavors of Dynamic Local PV - Hostpath, Device, LVM, ZFS or Rawfile. For more information you can read [here](/docs/next/localpv.html)
```

Comment:
Need to add a full stop after the Local PV link.
"For more information you can read [here](/docs/next/localpv.html)."